bombbot.py

```diff
@@ -212,8 +212,12 @@ def cancel_bomb(bot, trigger):
     """
     target = trigger.group(3) or None
     if not target:
-        bot.reply(STRINGS['CANCEL_WHOM'])
-        return
+        for bomb in BOMBS:
+            if trigger.nick == BOMBS[bomb]['bomber']:
+                target = BOMBS[bomb]['target']
+                break
```

Comment:
I'm not sure if it's worth further complicating the logic to bail if multiple bombs are found planted by the caller. Bot admins are the only people who can possibly plant multiple bombs, and the time window is short (currently two minutes).

Tempted to hold off merging this just because it would be a good exercise to implement that, but it would also make the code a lot uglier without much real benefit. Alternatively, `BOMBS` could be converted to a sort of LIFO queue by using `deque()`…but that seems like so much overkill. The only drawback to leaving it as-is? Once in a while, a bot admin might place multiple bombs, then `.bombcancel` the wrong one by leaving out the argument.